src/main/scala/org/codeoverflow/chatoverflow/requirement/parameter/ListParameterImpl.scala

```diff
@@ -0,0 +1,50 @@
+package org.codeoverflow.chatoverflow.requirement.parameter
+
+import java.util.Collections
+import java.util.List
+
+import collection.JavaConverters._
+import org.codeoverflow.chatoverflow.api.io.parameter.ListParameter
+import org.codeoverflow.chatoverflow.registry.Impl
+
+import scala.collection.mutable.ListBuffer
+
+/**
+  * A parameter holding a List<String> value.
+  */
+@Impl(impl = classOf[ListParameter])
+class ListParameterImpl extends ListParameter {
+  private var value: List[String] = null
+
+  override def getType: Class[List[String]] = classOf[List[String]]
+
+  override def serialize(): String = {
+    var out = ""
+    for (part <- value.toArray()) {
+      if (out == "") {
+        out = out + part
+      } else {
+        out = out + "," + part
+      }
+    }
+    out
```

Comment:
what about `value.asScala.mkString(",")` instead of this abomination?
(plus an import of `scala.collection.JavaConverters._`)